go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/segmentio/analytics-go
+
+go 1.14
+
+require (
+	github.com/segmentio/backo-go v0.0.0-20200129164019-23eae7c10bd3
+	github.com/segmentio/conf v1.2.0
```

Comment:
Having this package imported also brings lots of other dependencies, I wonder if it would be worth revisiting the code to remove this dependency.

Reply:
I'll take a look!